modules/installation-aws-regions-with-no-ami.adoc

```diff
@@ -1,26 +1,29 @@
 // Module included in the following assemblies:
 //
 // * installing/installing_aws/installing-aws-china.adoc
-// * installing/installing_aws/installing-aws-government-region.adoc
 // * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-aws-secret-region.adoc
 
 ifeval::["{context}" == "installing-aws-china-region"]
 :aws-china:
 endif::[]
-ifeval::["{context}" == "installing-aws-government-region"]
-:aws-gov:
+ifeval::["{context}" == "installing-aws-secret-region"]
+:aws-secret:
 endif::[]
+// ifeval::["{context}" == "installing-aws-government-region"]
+// :aws-gov:
+// endif::[]
 
 [id="installation-aws-regions-with-no-ami_{context}"]
-ifndef::aws-china,aws-gov[]
+ifndef::aws-china,aws-secret[]
 = AWS regions without a published {op-system} AMI
-endif::aws-china,aws-gov[]
+endif::aws-china,aws-secret[]
 
-ifdef::aws-china,aws-gov[]
-= Installation requirments
-endif::aws-china,aws-gov[]
+ifdef::aws-china,aws-secret[]
+= Installation requirements
+endif::aws-china,aws-secret[]
 
-ifndef::aws-china,aws-gov[]
+ifndef::aws-china,aws-secret[]
 You can deploy an {product-title} cluster to Amazon Web Services (AWS) regions
```

Comment:
@mjpytlak I do not see this section in the live doc.

Reply:
@yunjiang29 The content appears as Installation requirements and appears in the preview doc at https://deploy-preview-39769--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-secret-region#installation-aws-regions-with-no-ami_installing-aws-secret-region